### DISCUSSION TITLE
What's the best way to use x-model with both .fill and .debounce?

### DISCUSSION THREAD

**Discussion by bb** [alpinejs/alpine#4102] [7. Bugs (archived)]
What's the best way to use x-model with both .fill and .debounce? Using both together doesn't seem to work, the field is cleared.

**Comment by ekwoka** [comment]
Probably better to explain this as a bug (we talked about it on discord).

That the .debounce prevents the .fill from applying properly, instead resulting in the null being applied.

**Reply by bb** [reply]
Yeah, you're right.
I wondered why you asked for a discussion instead of an issue if you it's considered a bug. But thanks for the fix you made in PR #4103.

**Reply by ekwoka** [reply]
Alpine doesn't have issues. Only discussions :)

**Reply by bb** [reply]
I like that only linguistically. 😉
